staging-versions.ini

```diff
@@ -12,3 +12,4 @@ liquid-nextcloud = liquidinvestigations/liquid-nextcloud:0.2.3
 rocketchat = rocketchat/rocket.chat:3.9.1
 rocketchat-mongo = mongo:4.4
 pdf-preview = thecodingmachine/gotenberg:6
+image-classification = liquidinvestigations/image-classification-service:master
```

Comment:
Same as above: please tag a fixed version of this repo and put it here instead of `:master`